From 8c49b408b75c74da5aa980cf94b9e040c0d3b72e Mon Sep 17 00:00:00 2001
From: MisterZtr <andey2004586@gmail.com>
Date: Fri Oct 10 22:06:21 2025 +0300
Subject: fingerprint: Disable headers

diff --git a/fingerprint/Android.bp b/fingerprint/Android.bp
index 64d3a15..93a7e0a 100644
--- a/fingerprint/Android.bp
+++ b/fingerprint/Android.bp
@@ -10,7 +10,4 @@ cc_library_static {
     include_dirs: [
         "frameworks/native/services/surfaceflinger/CompositionEngine/include",
     ],
-    header_libs: [
-        "generated_kernel_headers",
-    ],
 }
