From 2f9861d0f6d8697aa0a468fd608bfee8e20d58d9 Mon Sep 17 00:00:00 2001
From: MisterZtr <andey2004586@gmail.com>
Date: Wed Oct 22 21:46:56 2025 +0300
Subject: build-Prefer-Etar-over-Calendar
    
    Change-Id: Ibe11835bbb8f61877d885c43dab95d854ed06023

diff --git a/target/product/gsi/Android.bp b/target/product/gsi/Android.bp
index 9063cdcf99..fda4f5cd4a 100644
--- a/target/product/gsi/Android.bp
+++ b/target/product/gsi/Android.bp
@@ -173,7 +173,7 @@ android_system_image {
         "EmergencyInfo",
 
         // handheld_product
-        "Calendar",
+        "Etar",
         "Contacts",
         "DeskClock",
         "Gallery2",
