From 4f2df120b37db31f0bb9d735051c5ca1b08dba6d Mon Sep 17 00:00:00 2001
From: MisterZtr <andey2004586@gmail.com>
Date: Fri Oct 10 22:02:41 2025 +0300
Subject: lineage: use own config

diff --git a/lineage.mk b/lineage.mk
index 172bb01..deb95c7 100644
--- a/lineage.mk
+++ b/lineage.mk
@@ -1,3 +1,21 @@
+$(call inherit-product, vendor/lineage/config/common_full_tablet.mk)
 $(call inherit-product, vendor/lineage/config/common_full_phone.mk)
--include vendor/lineage/build/core/config.mk
--include vendor/lineage/build/core/apicheck.mk
+$(call inherit-product, vendor/lineage/config/BoardConfigSoong.mk)
+$(call inherit-product, vendor/lineage/config/BoardConfigLineage.mk)
+$(call inherit-product, device/lineage/sepolicy/common/sepolicy.mk)
+
+# Disable adb secure
+WITH_ADB_INSECURE := true
+
+# Bootanimation
+TARGET_BOOT_ANIMATION_RES := 1080
+
+# Face unlock
+TARGET_FACE_UNLOCK_SUPPORTED := true
+PRODUCT_PACKAGES += FaceUnlock
+
+# APN
+PRODUCT_PACKAGES += apns-conf.xml
+
+#Oplus fingerprint
+$(call soong_config_set,surfaceflinger,udfps_lib,//hardware/oplus:libudfps_extension.oplus)
