From a0917e0a4413ead6eef7b8bb0d4f61cd4e14b9f1 Mon Sep 17 00:00:00 2001
From: MisterZtr <andey2004586@gmail.com>
Date: Wed Oct 22 21:43:32 2025 +0300
Subject: Fix build error

diff --git a/base.mk b/base.mk
index 553d5a5..d916095 100644
--- a/base.mk
+++ b/base.mk
@@ -33,7 +33,6 @@ PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
     ro.build.version.all_codenames=$(PLATFORM_VERSION_ALL_CODENAMES) \
     ro.build.version.release=$(PLATFORM_VERSION) \
     ro.build.version.security_patch=$(PLATFORM_SECURITY_PATCH) \
-    ro.adb.secure=0 \
     ro.logd.auditd=true \
     ro.logd.kernel=true \
 
@@ -195,9 +195,7 @@ PRODUCT_PACKAGES += \
     android.hardware.biometrics.fingerprint@2.1-service.oplus.compat \
 
 PRODUCT_PACKAGES += \
-    vr_hwc \
     curl \
-    healthd \
 
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
     debug.fdsan=warn_once \
     
diff --git a/system.prop b/system.prop
index 09ad957..c7a6b9a 100644
--- a/system.prop
+++ b/system.prop
@@ -1,4 +1,3 @@
-ro.adb.secure=0
 #Some devices have sdcardfs kernel panicing on 8.0
 #Disable for everyone for the moment
 ro.sys.sdcardfs=0

